Tools/machines/perlmutter-nersc/perlmutter_gpu.sbatch

```diff
@@ -24,9 +24,21 @@
 #SBATCH -o WarpX.o%j
 #SBATCH -e WarpX.e%j
 
-# executable & inputs file or python interpreter & PICMI script here
-EXE=./warpx
-INPUTS=inputs
+# PICMI Python script or executable?
+USE_PICMI=true
```

Comment:
This is a good idea. Maybe set `USE_PICMI=false` here so the default behavior is the same as previously.